index.html

```diff
@@ -16,5 +16,166 @@
 <body>
     <!-- load javascript after loading all html content -->
     <script src="script/script.js"></script>
```

Comment:
Make sure the script is at the end of the `body` tag. This way, all of the content is loaded when the HTML loads.